katas/main/src/pokerhands/valeria/.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Hi @svaleria , please remove `.vscode` folder and its files for now, they might conflict with custom configurations of the rest of the team